webapp/src/shared/components/Row/Row.css

```diff
@@ -39,3 +39,45 @@
   margin-left: auto;
   grid-column: 3/4;
 }
+
+.scorerow {
```

Comment:
suggestion: We could move this part to `ScoreRow.css` to self-contain the `ScoreRow` component styling.

Reply:
I will change it